src/pipecat/processors/markdown_remover.py

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright (c) 2024, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+import re
+
+from markdown import Markdown
```

Comment:
should we add the markdown dependency? if it's not big it could go into the main ones.

Reply:
The source is only 357kb, so it should be small enough.